genisys/modules/nat.py

```diff
@@ -68,7 +68,8 @@ def install_location(self: Self) -> Path:
     # end install_location
 
     def setup_commands(self: Self) -> Union[List[str], List[List[str]]]:
-        return ["sudo netfilter-persistent save", "sudo netfilter-persistent reload", "sudo systemctl enable iptables", "sudo systemctl start iptables"]
+
+        return ["iptables-restore < " + self.IPV4_DIR, "netfilter-persistent reload", "systemctl enable iptables", "systemctl start iptables", "sysctl -p"]
```

Comment:
For security reasons, these "commands" need to be ran as subprocesses, not in a shell. For this reason, shell redirection, ie. `<` will probably not work. If there is not another syntax for `iptables-restore` we will have to rethink this `setup_commands` function to allow for changing the value of `stdin` for the subprocess.

Reply:
Addressed in commit a22943b